packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -0,0 +1,34 @@
+# React Scheduler - Grouping
+
+The React Scheduler allows you to group appointments by their [resources](../reference/resources.md). The group names are displayed by [GroupingPanel](../reference/grouping-panel.md) plugin.
+
+To change the group an appointment belongs to users can edit a corresponding appointment through the [AppointmentForm](../reference/appointment-form.md) or by drag-dropping the appointment with the help of [DragDropProvider](../reference/drag-drop-provider.md) plugin.
+
+## Related Plugins
+
+- [Resources](../reference/resources.md) - configures resources
+- [GroupingState](../reference/grouping-state.md) - manages the grouping state
+- [IntegratedGrouping](../reference/integrated-grouping.md) - performs built-in grouping
+- [GroupingPanel](../reference/grouping-panel.md) - renders the Grouping Panel
```

Comment:
`GroupingPanel - renders the Grouping Panel` - good explanation 😂
Group Panel displays grouped resource titles